src/index.ts

```diff
@@ -20,49 +21,60 @@ const run = async (): Promise<void> => {
       );
     }
 
-    let prData = await getData(authToken);
+    const coverageConfiguration: PathProperties[] = await getConfiguration();
 
-    if (url && authToken) {
-      try {
-        prData = await sendPullRequestData(url, prData);
-      } catch (error) {
-        console.log(`${error}, Could not send data, printing comment`);
-      }
-    }
+    await Promise.all(
+      coverageConfiguration.map(async (configuration) => {
```

Comment:
Try catch this or add some check, this split can fail

Reply:
wouldn't the split for the path always succeed though? Because we would always have a string for the path so the `.split` would always be valid.